Objects/TI4_Helpers/TI4_ExploreHelper.ttslua

```diff
@@ -933,6 +1007,573 @@ function AttachLib._layoutDecals(decals)
+    local planet = false
+    for _, systemPlanet in ipairs(system.planets or {}) do
+        if planetName == systemPlanet.name then
+            planet = systemPlanet
+            break
+        end
+    end
+    assert(planet)
+
+    if planet.legendary then
+        printToAll('Cannot use Stellar Converter on a Legendary planet.', 'Yellow')
+        return false
+    end
+
+    -- Detect if we're attaching to planet that's already acted on (eg. onLoad after using Stellar Converter)
+    -- Detection options...none are perfect.
+    --  Is the token locked at hover height?
+    --  Is the planet card in the Purge bag?
+    --  Token is identical on each side, so faceup could be pre-purge, facedown could be post-purge.
+    local purgeBag = _getByName('Purge Bag')
+    if purgeBag then
+        for _, entry in ipairs(purgeBag.getObjects()) do
+            -- Planet has been purged. Token should just anchor down.
+            if entry.name == planetName then
+                -- Save planet state (onLoad will miss any attachments that are already in the Purge bag. Players will have to handle.)
+                system._stellarConverterPlanet = copyTable(planet)
+                -- Clear planet stats
+                planet.resources = 0
+                planet.influence = 0
+                planet.tech = nil
+                planet._attachments = nil
+
+                -- Place token directly on planet, and lock it
+                attachTokenObject.setLock(false)
+                local planetPosition = systemObject.positionToWorld(planet.position)
+                local tokenPosition = { x = planetPosition.x, y = planetPosition.y + 0.5, z = planetPosition.z }
+                local collide = false
+                local fast = true
+                attachTokenObject.setPositionSmooth(tokenPosition, collide, fast)
+
+                local function condition()
+                    if attachTokenObject.isSmoothMoving() then
+                        return false
+                    end
+                    if not attachTokenObject.resting then
+                        return false
+                    end
+                    return true
+                end
+                local function action()
+                    attachTokenObject.setLock(true)
+                end
+                local function timeout() end
+                Wait.condition(action, condition, 3, timeout)
+
+                -- Return modified system. No further action required.
+                return system
+            end
+        end
+    end
+
+    -- 'Hover' over target planet (don't disrupt units, tokens, etc.)
+    local planetPosition = systemObject.positionToWorld(planet.position)
+    local tokenPosition = { x = planetPosition.x, y = planetPosition.y + 5, z = planetPosition.z }
+    local collide = false
+    local fast = true
+    attachTokenObject.setPositionSmooth(tokenPosition, collide, fast)
+
+    -- Lock in mid-air as soon as moving finishes.
```

Comment:
I'm not sure about locking in mid-air, as a general style everything in right click menus should be optional/convenience.

Reply:
So, doing it this way got around the following:
- It's a big token, and looks goofy placed directly on top of smaller attachments (which puts it at an angle)
- I didn't want dropping it alone to be the trigger to blow stuff up. Too easy to misclick or misplace (particularly in a 3 planet system, which is tight).

I agree that my solution doesn't feel "finished"